environments/xhibit-portal.json

```diff
@@ -10,6 +10,15 @@
         }
       ]
     },
+    {
```

Comment:
This file should be in a separate PR and merged in first to ensure the AWS account is created before any networking takes place.

Reply:
That's also been cleaned up as it's not relevant for this stage - thanks for the comment!